app/src/main/res/layout/fragment_key_gear.xml

```diff
@@ -66,7 +66,9 @@
             android:id="@+id/items"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/base_margin_double"
```

Comment:
Does this play correctly with the `ItemDecoration` we're using for this `RecyclerView`? 🙂